src/components/release-date-picker/release-date-picker.tsx

```diff
@@ -0,0 +1,37 @@
+import { Input } from 'components/input';
+import React, { FC, forwardRef, useState } from 'react';
+import DatePicker from 'react-datepicker';
+import { ReleaseDatePickerIcon, ReleaseDatePickerWrapper } from './styles';
+
+export const ReleaseDatePicker: FC = () => {
+  const [startDate, setStartDate] = useState(null);
+
+  const DatePickerCustomInput = forwardRef(
```

Comment:
you basically have a component declared inside another component. Move this to separate file